src/common/utils.cpp

```diff
@@ -33,6 +33,8 @@
 #	include <intrin.h>
 #endif
 
+#define MAX_ITEM_SIGNATURE_LENGTH 15
```

Comment:
it's not the max item signature length, it's just the maximum character name length, isn't it? so maybe this should be somewhere related to that and renamed?

also, please favor constexpr over `define`

Reply:
like constexpr size_t GetPlayerNameLengt() { return 15; } ? I feel bad that I've never used constexpr before. It's still compile time from what I understand?

Switching that over and cleaning up other hardcoded places that use 15 for player name length. A lot of them are redundant because CCharEntity::SetName(...) already limits the number of characters that get copied.

Reply:
just `constexpr size_t max_name_length = 15;` should suffice